workflow/rules/frag_to_tagAlign.smk

```diff
@@ -18,18 +18,11 @@ rule frag_to_tagAlign:
 	shell:
```

Comment:
```suggestion
	threads: 5
	shell:
```

I think something like this would work after looking at https://snakemake.readthedocs.io/en/v7.22.0/executing/cluster.html#ordinary-smp-jobs

Reply:
and threads = CPUs? how do we decide how many??

Reply:
according to the link, yes. 

More is better, but this is a reasonable amount that should get the job assigned quickly. You can increase it as you see fit